### DISCUSSION TITLE
Extra Co-op options

### DISCUSSION THREAD

**Discussion by slambo311** [BEEmod/BEE2-items#4126] [Feature Suggestions]
There could be separate spawn points for each player (So that they _have_ to go into their own part of the test), changing their player models individually (Could be useful combined with the spawn points), disabling partner view, etc.

**Comment by SimpleFoxOfficial** [comment]
and Portal 1 style support lol :)

**Reply by vrad-exe** [reply]
There's already P1 Coop, it's been there since BEE2.2.

**Reply by SimpleFoxOfficial** [reply]
it's crashes. Always.

**Reply by vrad-exe** [reply]
It does not crash "always". What items are in your map, and what stylevars are you using?

**Comment by vrad-exe** [comment]
We don't want to force each player to do a specific part of the test, there's a reason the separated checkpoint allows you to choose which side you go on, and also why there are relatively few player-specific items currently. See #3317 for one of the plans for that.

I'm not sure if it's possible to disable remote view, but why would you want to do that in a puzzlemaker map anyway?

**Reply by slambo311** [reply]
Honestly, I don't know about partner view. Just thought maybe some people could make a test where players aren't supposed to see each other's screen.
